### PR TITLE
Update basic_indicator.py

### PR DESCRIPTION
**Pull request by tsv19su254052**
Подправил

### CODE REVIEW

**Comment by NewYongJun** [comment]
Hello.

**Comment by webcat051** [comment]
Hello.

File: trading_bot/indicators/basic_indicator.py [hunk 1]
```diff
@@ -192,18 +164,17 @@ def __init__(self, period: int = 14) -> None:
         if period < 0:
             raise ValueError("period cannot be lesser than 0")
         period = int(period)
-        BasicIndicator.__init__(
-            self, "Lowest Low (" + str(period) + ")", period)
+        BasicIndicator.__init__(self,
```
Comment (webcat051) [inline trading_bot/indicators/basic_indicator.py:167]: why did you put Enter on this line.
Could you change?

File: trading_bot/indicators/basic_indicator.py [hunk 2]
```diff
@@ -108,39 +90,36 @@ def __init__(self, period: int = 14) -> None:
         if period < 0:
             raise ValueError("period cannot be lesser than 0")
         period = int(period)
-        BasicIndicator.__init__(
-            self, "Highest High (" + str(period) + ")", period)
+        BasicIndicator.__init__(self,
```
Comment (webcat051) [inline trading_bot/indicators/basic_indicator.py:93]: and Could you please change this line?

**Review by webcat051 — CHANGES_REQUESTED** [review]
Hello.
Thanks for your use my repository  and request push.
I did check all code.
Could you please change about my request?
Thanks.

**Pull request merged**